### ISSUE TITLE
After install, add a warning that the *.sandcats.io subdomain takes time

### ISSUE THREAD

**Issue by GreatScott** [sandstorm-io/sandstorm#618]
*super minor*

Upon finishing the installation, there is the note:

"Setup complete. You should configure the site at:
  http://<subdomain>.sandcats.io:6080/admin/settings/<key>"

It took ~20-30min for the URL to work, however. By then the generated admin token is invalid and a new one must be generated. Might suggest just leaving a warning instead, as the splash page already lists the command needed to generate the token once it's up.

**Comment by paulproteus** [comment]
Sad! It shouldn't take​ that long. Hmm.

**Comment by GreatScott** [comment]
Actually, wasn't timing it precisely - sorry (terrible issue name). The issue is more that the URL doesn't work immediately after install, but the message *seems* to indicate otherwise.

**Comment by paulproteus** [comment]
Yeah! For what it's worth, when I designed the sandcats.io service, I built
it so that it should be possible for these names to work immediately. It's
good to know that they sometimes don't work immediately, though. We should
adjust the text so people are prepared for that possibility.

**Comment by kentonv** [comment]
One case where the URL may not work immediately is if the user has done a DNS lookup for the URL ahead of time -- perhaps to check if it is available -- and thus ended up with "not found" in their cache. Maybe we can fix this by setting the TTL for "not found" queries to something small?

**Comment by GreatScott** [comment]
I'll dive in to add that I didn't try any sort of lookup / try the URL in the browser in advance. The timing delay was on the order minutes ( > 5 ). Ah BTW, this was from a DO droplet in NYC.
